source/actions3/actions/AMControlMoveAction3.cpp

```diff
@@ -85,15 +114,21 @@ void AMControlMoveAction3::startImplementation()
 	// remember the start position:
 	startPosition_ = control_->toInfo();
 
-	// start the move:
-	int failureExplanation;
+	attempts_ = 1;
+
+	// determine the absolute destination
 	if(controlMoveInfo()->isRelativeMove() && controlMoveInfo()->isRelativeFromSetpoint())
-		failureExplanation = control_->moveRelative(setpoint.value(), AMControl::RelativeMoveFromSetpoint);
+		destination_ = control_->setpoint() + setpoint.value();
```

Comment:
You did this change after I left, right?  Does this still work properly if a control move action was inside of a loop doing a relative move 10 times?